config/locales/server.es.yml

```diff
@@ -374,6 +375,7 @@ es:
           ¿Sabes que puedes agregar una encuesta en cualquier mensaje? Intenta usando el botón de engranaje <img src="/images/font-awesome-gear.png" width="16" height="16"> en el editor para **armar una encuesta**.
         not_found: |-
           Whoops! No hubo ninguna encuesta en tu respuesta.
+
           Usa el ícono de engranaje: <img src="/images/font-awesome-gear.png" width="16" height="16"> en el editor, o bien copia y pega esta encuesta en tu próxima respuesta:
```

Comment:
Check this out @ZogStriP 
Here (in english) is happened too: https://github.com/discourse/discourse-narrative-bot/blob/70f87390f0b123ebca552513cf843fc5f9e674c7/config/locales/server.en.yml#L404

The line 404 (in server.en.yml) is highlighted in green when It's text (nor a text-label for translation).
Can you check it? 

(I hope you can understand what I mean) :blush: